Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:alpine
```

Comment:
should we use golang:1.11 image here? Would match the golang version we are targeting.

Reply:
oh wait you compile it then add it into this container. Should you compile it in another container like golang:1.11 i mean.

Reply:
Yes I plan to add another target to compile it in a container. This target here is really for the IT testing - this target here is just a golang container that we launch the binary into to see if it works.